src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -313,6 +326,10 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 	private void updateFields() {
 		if (toolType.getSelectedItem() != null) {
+			blockingModel.setEnabled(true);
+			if (!toolType.getSelectedItem().equals("Shield") || (renderType.getSelectedItem() == null || renderType.getSelectedItem().getType() == Model.Type.BUILTIN))
+				blockingModel.setEnabled(false);
```

Comment:
If BUILTIN model type is selected, one would probably want to set blockingModel to normalBlocking before disabling it.

Otherwise one can select eg. JSON model and then switch to normal built-in model and the blockingModel field will get disabled on JSON model, which is wrong information for the user